extensions/adobe/experience/offer-management/capping-constraint.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/offer-management/capping-constraint",
+  "type": "object",
+  "title": "Capping Constraint",
+  "description": "Offer Capping Constraint",
+  "definitions": {
+    "capping-constraint": {
+      "type": "object",
+      "properties": {
+        "xdm:volume": {
+          "type": "integer",
+          "title": "Volume",
+          "description": "Total offer volume",
+          "multipleOf": 1,
+          "minimum": 1
+        },
+        "xdm:frequency": {
+          "type": "number",
+          "title": "Frequency",
+          "description": "Frequency per profile",
+          "multipleOf": 0.1,
+          "minimum": 0,
+          "maximum": 100
+        }
+      },
+      "required": ["xdm:volume", "xdm:frequency"]
```

Comment:
`xdm:volume` and `xdm:frequency` should not be required fields. It is possible that an offer just has one of the two.

Reply:
FIxed